<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node name="path_simulator" pkg="sparse_map_utils" type="pathSimulation.py" args="50 $(find sparse_map_utils)/results/run1.csv" output="screen">
    <remap from="move_make_plan" to="/move_base/make_plan"/>
    <remap from="sparse_make_plan" to="/sparse_map/make_plan"/>
    <remap from="static_map" to="truth_map"/>
  </node>
</launch>
